src/main/java/forestry/apiculture/ModuleApiculture.java

```diff
@@ -155,7 +155,9 @@ public class ModuleApiculture extends BlankForestryModule {
 
 	public static String beekeepingMode = "NORMAL";
 
-	public static int ticksPerBeeWorkCycle = 550;
+	// Previous value was 550 (~27 IRL seconds) - way too high. It's better IMO to scale down the production (or up) rather than have a very long tick rate and an enormous output at once
+	// Is set later from the config file, by default is set to 40 ticks
+	public static int ticksPerBeeWorkCycle;
```

Comment:
The default should be left at 550. A config option would be a better alternative, so that GregTech players can scale the work cycle ticks without affecting the rest of the playerbase. Keep in mind that reducing this number reduces the in-game lifespan of a bee. I also don't think the upper limit should be reduced.